conda_env/specs/remote.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""Remote spec using fsspec"""
+from functools import cached_property
+
+import fsspec
+from fsspec import available_protocols
+from fsspec.core import split_protocol
```

Comment:
This will require `fsspec` to be a required dependency of conda.  
If this is not desired the import of `RemoteSpec` could be delayed until needed in the __init__.py file so that the dependency is optional and only required if a remote spec is needed.

Reply:
I'm cool with that if it might get this going faster while the `conda env` effort continues